test/Mono.Linker.Tests.Cases/DataFlow/GenericParameterDataFlow.cs

```diff
@@ -68,25 +68,15 @@ static void TestGenericParameterFlowsToReturnValue ()
 		class TypeRequiresPublicFields<
 			[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicFields)] T>
 		{
-			[UnrecognizedReflectionAccessPattern (typeof (DataFlowTypeExtensions), nameof (DataFlowTypeExtensions.RequiresPublicMethods), new Type[] { typeof (Type) },
-				messageCode: "IL2087", message: new string[] {
-					nameof (T),
-					nameof (TypeRequiresPublicFields <T>),
-					nameof (DataFlowTypeExtensions.RequiresPublicMethods)
-				})]
+			[ExpectedWarning ("IL2087", "'" + nameof (T) + "'", nameof (TypeRequiresPublicFields<T>), nameof (DataFlowTypeExtensions.RequiresPublicMethods))]
```

Comment:
I think this file needs [ExpectedNoWarnings]

Reply:
I see the attribute already there - maybe I'm missing something?

Reply:
I see you didnt include it, it was already there. Sorry my mistake